polaris-react/src/components/KeyboardKey/KeyboardKey.stories.tsx

```diff
@@ -7,5 +7,9 @@ export default {
 } as ComponentMeta<typeof KeyboardKey>;
 
 export function Default() {
```

Comment:
can we maybe add multiple types/scenarios of keyboard shortcuts in the stories? maybe one for save on different platforms (cmd + S / ctrl + S), exit (BYE)